pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -0,0 +1,71 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <PackageVersion>2.1.0</PackageVersion>
```

Comment:
@terrajobst @danmosemsft should this version be 2.0.0 as well?

Reply:
Oh we might want to make this version 2.0.0 as well as it is a new package since we renamed it. I'll wait for the answer though.

Reply:
Unless there's a reason it can't be?

Reply:
I'm for version 2.0